src/OpenTransitions.jl

```diff
@@ -0,0 +1,44 @@
+""" Types to compose Petri nets along transitions.
+"""
+
+module OpenTransitions
+
+using AlgebraicPetri
```

Comment:
```suggestion
using AlgebraicPetri
using AlgebraicPetri: LabelledPetriNetUntyped
```

We can access `LabelledPetriNetUntyped` like this rather than exporting it.